src/Fade.js

```diff
@@ -2,87 +2,77 @@ import React from 'react';
 import Transition from './Transition';
 
 class Fade extends React.Component {
-
-  constructor(props, context){
-    super(props, context);
-  }
-
   render() {
     return (
       <Transition
         {...this.props}
-        in={this.props.in}
         className='fade'
         enteredClassName='in'
         enteringClassName='in'
       >
-        { this.props.children }
+        {this.props.children}
       </Transition>
     );
   }
 }
 
+// Explicitly copied from Transition for doc generation.
```

Comment:
I have been thinking about this...if we had a standard omit, we could teach the metadata about it. Right now tho it will probably choke on anything that isn't an object literal, so you'd need to use a spread like: `..._.omit(blah, ['bah'])`